aggsender/db/migrations/0003.sql

```diff
@@ -0,0 +1,11 @@
+-- +migrate Down
+ALTER TABLE certificate_info DROP COLUMN cert_type;
+ALTER TABLE certificate_info DROP COLUMN cert_source;
+ALTER TABLE certificate_info_history DROP COLUMN cert_type;
+ALTER TABLE certificate_info_history DROP COLUMN cert_source;
+
+-- +migrate Up
+ALTER TABLE certificate_info ADD COLUMN cert_type VARCHAR DEFAULT "";
+ALTER TABLE certificate_info ADD COLUMN cert_source VARCHAR DEFAULT "";
```

Comment:
Would this make sense for already existing certificates in the db?

```suggestion
ALTER TABLE certificate_info ADD COLUMN cert_source VARCHAR DEFAULT "local";
```

Reply:
No, the previous certificate we don't known the source